examples/step-3/doc/intro.dox

```diff
@@ -102,19 +109,24 @@ where the matrix $A$ and the right hand side $F$ are defined as
   \\
   F_i &= (\varphi_i, f).
 @f}
+
+
+<h3> Multiplying with a test function from the left or from the right? </h3>
+
 Before we move on with describing how these quantities can be computed, note
 that if we had multiplied the original equation from the <i>right</i> by a
 test function rather than from the left, then we would have obtained a linear
 system of the form
 @f{align*}
-  U^T A = F
+  U^T A = F^T
```

Comment:
Nice catch!

Reply:
It wasn't wrong before (the text explicitly said that $F$ is a row vector in this case), but I felt that it was easier to read this way.